source/h1_connection.c

```diff
@@ -1672,6 +1941,11 @@ static int s_handler_shutdown(
             s_stream_complete(AWS_CONTAINER_OF(node, struct h1_stream, node), stream_error_code);
         }
 
+        while (!aws_linked_list_empty(&connection->thread_data.waiting_stream_list)) {
```

Comment:
If we need a new error code to inform user the circumstance that the stream already has a response, but those ones in front of it are not ready, and cause it failed to send the response